packages/dd-trace/test/plugins/helpers.js

```diff
@@ -33,7 +34,7 @@ function expectSomeSpan (agent, expected, timeout) {
     const error = scoredErrors.sort((a, b) => a.score - b.score)[0].err
     // We'll append all the spans to this error message so it's visible in test
     // output.
-    error.message += '\n\nCandidate Traces:\n' + JSON.stringify(traces, null, 2)
+    error.message += '\n\nCandidate Traces:\n' + JSON.stringify(traces, replacer, 2)
```

Comment:
Does the error need to be JSON formatted? If not, lets use `util.inspect()`

```suggestion
    error.message += '\n\nCandidate Traces:\n' + util.inspect(traces)
```